webapp/src/components/common/DataGridViewer.tsx

```diff
@@ -20,26 +20,34 @@ import {
   type TextCell,
 } from "@glideapps/glide-data-grid";
 import { useMemo } from "react";
-import { formatGridNumber } from "./Matrix/shared/utils";
-import DataGrid from "@/components/common/DataGrid";
+import { formatGridNumber, isNumericString } from "./Matrix/shared/utils";
+import DataGrid, { type RowMarkers } from "@/components/common/DataGrid";
 
 type CellValue = number | string;
 
 interface DataGridViewerProps {
   data: CellValue[][];
   columns: GridColumn[];
+  freezeColumns?: number;
+  rowMarkers?: RowMarkers;
 }
 
-function DataGridViewer({ data, columns }: DataGridViewerProps) {
+function DataGridViewer({
+  data,
+  columns,
+  freezeColumns = 0,
```

Comment:
`freezeColumns` is already 0 by default and rowMarkers is already "none" by default.
Don't set them here allows to keep the same default values on the two components.

Reply:
its intended to be set here